src/data_cleaning.py

```diff
@@ -191,12 +172,27 @@ def generate_subplant_ids(start_year, end_year, cems_ids):
     )
 
 
+def manually_update_subplant_id(subplant_crosswalk):
+    """
+    This function corrects subplant mappings not caught by update_subplant_id.
+
+    This is temporary until the pudl subplant crosswalk includes boiler-generator id matches.
```

Comment:
Huh! Did this end up being the root issue of the GTN conversion problems with this plant?

Reply:
It did. It's because a missing association between the boiler and generator, so something to fix in pudl in the future.